torch/onnx/_internal/fx/passes/fx_to_onnxscript.py

```diff
@@ -476,21 +436,35 @@ def _export_fx_node_to_onnxscript(
         assert (
             output is not None
         ), f"Node creates None with target={node.target}, name={node.name}, args={onnx_args}, kwargs={onnx_kwargs}"
-        # TODO(justinchuby): Add diagnostic information.
-        # Assign type and shape obtained from FakeTensorProp.
+        # Assign type and shape from fx graph.
         _fill_tensor_meta(output, node.name, node.meta["val"])
         # One fx node could produce multiple outputs (e.g., tuple of tensors); in
         # that case, v is a tuple of TorchScriptTensors.
         assert isinstance(output, (graph_building.TorchScriptTensor, tuple)), type(
             output
         )
-        if options.op_level_debug:
-            # FIXME(titaiwang): Refactor me with
-            # https://github.com/microsoft/onnx-script/issues/393
-            (torch_args, torch_kwargs) = _wrap_fx_args_as_torch_args(
-                complete_args, complete_kwargs
+        # NOTE(titaiwang): We bypass two kinds of ops as it's not meaningful to
+        # validate them with op level debug.
+        # 1. aten::sym_size: The op is simply get item from a list of tensors.
+        # 2. BuiltinFunction: It doesn't supported tensor
+        if (
+            options.op_level_debug
+            and node.target != torch.ops.aten.sym_size
+            and not isinstance(node.target, types.BuiltinFunctionType)
+        ):
+            node_with_static_shape = node.meta["node_with_static_shape"]
+            (
+                node_with_fixed_shape_args,
+                node_with_fixed_shape_kwargs,
+            ) = _fill_in_default_kwargs(node_with_static_shape)
+            # TODO(titaiwang): Why some ops failed on op-level validation, but
```

Comment:
gap between eager vs export-and-execute?

Reply:
Legacy. I will remove it..